### DISCUSSION TITLE
Cannot create distribution list - no button

### DISCUSSION THREAD

**Discussion by ghost** [modoboa/modoboa#1813] [Q&A]
# Impacted versions

* Modoboa: 1.14.0
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce
- Install Modoboa
- Configure domain, DKIM, SPF, DMARC etc.

# Current behavior
- I can create user accounts and aliases,  but no distribution lists or forwards

# Expected behavior
- Create distribution lists and forwards

# Video/Screenshot link (optional)
https://imgur.com/zQ6ylKe

**Comment by tonioo** [comment]
@mucberg distribution lists are not supported and forwards are just aliases using account addresses as source.

**Comment by ghost** [comment]
Thank you for your reply. Can you explain why this feature is not supported? I have a few scenarios where distribution lists come in handy; instead i would have to create and monitor additional accounts.

**Comment by almereyda** [comment]
Distribution lists are a special case of an account having aliases defined parallely, which point at the various recipients.

**Comment by tonioo** [comment]
Are we talking about distribution lists or mailing lists? @mucberg What is your exact need?

**Comment by ghost** [comment]
As described, having distribution lists would be nice.
